pillar.example

```diff
@@ -11,6 +11,7 @@ openldap:
   group: 'openldap'
   slapd_services: 'ldap://127.0.0.1:389/ ldaps:/// ldapi:///'
   slapd_options: '-4'  # add other options here if needed
+  slapd_include_dir: /etc/ldap/include
   includes:
     my_include_file: |
       content
```

Comment:
Would you mind changing this (and the lines below) to a small, sample include file that actually works?  Then we can get this part added to the tests later on.